web/src/test/resources/com/epam/rft/atsy/cucumber/Login.feature

```diff
@@ -43,4 +43,13 @@ Feature: Login in
   Scenario: username inputfield is automatically in focus
 
     Given the user is on the login page
-    Then the username field is in focus
\ No newline at end of file
+    Then the username field is in focus
+
+  Scenario: user redirected to welcome page if already signed in
```

Comment:
The description of the scenario is not clear.

Enter the credentials, then reload the page (possibly losing the entered credentials), then submitting the login form and checking if the use was able to log in.

It is a mess, and should be rethink.